media-video/syncplay/syncplay-1.4.0_rc4.ebuild

```diff
@@ -30,7 +30,13 @@ RDEPEND="${PYTHON_DEPS}
 
 S="${WORKDIR}/${PN}-${MY_PV}"
 
-PATCHES=( "${FILESDIR}/syncplay-1.4.0-rc2-fix-makefile.patch" )
+src_prepare() {
+	use gui ||
```

Comment:
don't do these things (beware of the subshell). Instead:
```
if use gui; then
	sed -i 's/"noGui": False,/"noGui": True,/' \
		syncplay/ui/ConfigurationGetter.py ||
		die "Failed to patch ConfigurationGetter.py"
fi
```
and preferably do it after `default`, not before